InterfaceLib/FakeResources.c

```diff
@@ -450,9 +452,9 @@ void	FakeUpdateResFile( int16_t inFileRefNum )
 			resMapOffset += 4;
 			fwrite( *currMap->typeList[x].resourceList[y].resourceHandle, 1, theSize, currMap->fileDescriptor );
 			resMapOffset += theSize;
-			
-			refListSize += currMap->typeList[x].numberOfResourcesOfType * (2 + 2 + 1 + 3 + 4);
 		}
+
+		refListSize += currMap->typeList[x].numberOfResourcesOfType * (2 + 2 + 1 + 3 + 4);
```

Comment:
I know all the magic numbers are from the original code, but any chance we could get consts for these ?

Reply:
I'm not sure I'd want to do it for every little size, but I sort of considered it for repeating clusters like this. I can do that for sure, and I'll think about the former.